File: PoznajPilkarza/Entities/Player.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace PoznajPilkarza.Enitites
{
    public class Player
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int PlayerId { get; set; }
        public int ShirtNumber { get; set; }
        public string Name { get; set; }

        public string Surname { get; set; }
        public int NationalityId { get; set; }
        public virtual Nationality Nationality { get; set; }
        public int PositionId { get; set; }
        public virtual Position Position { get; set; }
        public int Height { get; set; }

        public int Weight { get; set; }
        public DateTime DateOfBirth { get; set; }

        public int TeamId { get; set; }
        public virtual Team Team { get; set; }

        public string PngImage { get; set; }

        public string Description { get; set; }

        public string WikiLink { get; set; }
    }
}

File: PoznajPilkarza/ClientApp/src/app/models/player.ts
export interface IPlayer {
    name: string;
    surname: string;
    shirtNumber: number;
    nationalityName: string;
    dateOfBirth: Date;
    positionName: string;
    height: number;
    weight: number;
    teamName: string;
    leagueName: string;
    nationalLeagueName: string;
    flagNational: string;
    wikiLink: string;
    description: string;
}
export class Player implements IPlayer {
    name: string; surname: string;
    shirtNumber: number;
    nationalityName: string;
    dateOfBirth: Date;
    positionName: string;
    height: number;
    weight: number;
    teamName: string;
    leagueName: string;
    nationalLeagueName: string;
    flagNational: string;
    wikiLink: string;
    description: string;
    /**
     *
     */
    constructor(name: string) {
        this.name = name;

    }

}
